### ISSUE TITLE
Possibility to scale series

### ISSUE THREAD

**Issue by Mickael-Martin** [goat1000/SVGGraph#82]
hello,

I search a way to transform/scale one or more series to obtain graph like that : 

![BrsOZzaCQAAyqBZ](https://github.com/goat1000/SVGGraph/assets/1871792/c53ec5de-0327-40fa-b743-9e148c24307d)

![mailstat-daily](https://github.com/goat1000/SVGGraph/assets/1871792/98315ee9-65c8-4a7d-a1f6-f9ec3e307583)

I don't find any option to make that.

**Comment by goat1000** [comment]
I'm sorry, I don't understand what you are trying to do. The examples you gave look like line graphs with negative values.

**Comment by Mickael-Martin** [comment]
Hello,

No, it's positive values of Y negative axis. You can see legends in the second example, all values are positive. This feature is usefull to display bandwidth, for example : 

![image](https://github.com/goat1000/SVGGraph/assets/1871792/dfbb3097-6057-419b-aef8-917539e5464e)

**Comment by goat1000** [comment]
The easiest way to do that is to pre-process the values you pass to SVGGraph, and multiply all the values you want to be negative by -1.
```PHP
// make series at array positions 3 and 4 negative
$negative_index = [3, 4];

foreach($values as $key => $row) {
  foreach($negative_index as $index) {
    $values[$key][$index] = $row[$index] * -1;
  }
}
```

**Comment by Mickael-Martin** [comment]
Yes, I know that, but you got wrong values. The idea is an additional Y axis inverted.

**Comment by goat1000** [comment]
OK, I'll add it to the list of things to look at.

**Comment by Mickael-Martin** [comment]
Thank you very much : I want to use that to extend Zabbix widget (svggraph widget use your project, do you know ?) to get beautiful graphs for bandwidth.

**Comment by goat1000** [comment]
Are you sure that Zabbix uses SVGGraph? I can't see anything about it.

**Comment by Mickael-Martin** [comment]
Oh no, I was very wrong. I have skimmed through the PHP classes and linked to your project. I'm really sorry for this blunder...
And the worst part is that two of us made the connection (with a colleague)... So you can close this "issue" if you like.
I am frankly ashamed.
